tcp_check/tests/test_tcp_check.py

```diff
@@ -14,57 +14,31 @@ def aggregator():
     return aggregator
 
 
-@pytest.fixture
-def check():
-    return TCPCheck(CHECK_NAME, {}, {})
-
-
-@pytest.fixture
-def instance_ko():
-    return {
-        'host': '127.0.0.1',
-        'port': 65530,
-        'timeout': 1.5,
-        'name': 'DownService',
-        'tags': ["foo:bar"],
-    }
-
-
-@pytest.fixture
-def instance():
-    return {
-        'host': 'datadoghq.com',
-        'port': 80,
-        'timeout': 1.5,
-        'name': 'UpService',
-        'tags': ["foo:bar"]
-    }
-
-
-def test_down(aggregator, check, instance_ko):
+def test_down(aggregator, check):
     """
     Service expected to be down
     """
-    check.check(instance_ko)
+    check.check(deepcopy(common.INSTANCE_KO))
```

Comment:
Does the check mutate the instance?

Reply:
It does not, but I don't know if we should rely on that